src/frontend/src/components/Map.js

```diff
@@ -143,11 +156,39 @@ const Map = ({
   hullPoints,
   predictionHexagonIDs,
   annotationHexagonIDs,
-  onAddAnnotationHexagonIDs,
   hexResolution,
   taxonId,
+  onAddAnnotationHexagonIDs,
+  onAddAnnotationMultiSelect,
 }) => {
   console.log('Render map');
+
+  const [multipleAnnotationHexagonIDs, setMultipleAnnotationHexagonIDs] = useState(null);
+  const [isAddAnnotationMultiSelect, setIsAddAnnotationMultiSelect] = useState(true);
+
+  const handleCreated = (e) => {
+    try {
+      const layer = e.layer;
+      const polygonCoords = layer.getLatLngs()[0].map((latlng) => [latlng.lat, latlng.lng]);
+      var hexagonIds = h3.polyfill(polygonCoords, hexResolution);
+      // Add hexagons for each vertex of the polygon
+      polygonCoords.map((latlng) => hexagonIds.push(h3.geoToH3(latlng[0], latlng[1], hexResolution)));
+    // catch an error if the figure is not fully drawn
+    } catch (error) {
+      var hexagonIds = null;
```

Comment:
Maybe define hexagonIds outside and before try-catch initialize it to null, to resolve lint error

Reply:
in the current way, if you throw error after initializing hexagonIds I think it will crash it because you are re-initializing hexagonIds